### DISCUSSION TITLE
Question: Conquest's ASE interface module location

### DISCUSSION THREAD

**Discussion by kimrojas** [OrderN/CONQUEST-release#168] [General]
Hi

I'm currently trying to explore conquest.

The [documentation](https://conquest.readthedocs.io/en/latest/ext-tools.html#atomic-simulation-environment-ase) mentions that an ASE interface is available, however, I checked the current master branch of ASE and its still not available there. 

May I ask where it might be (if its already available)? 

Edit: I've also tried to use the 3.22.1 verison of ase with the `from ase.calculators.conquest import Conquest` but it simply results to `ModuleNotFoundError`. 

Thank you.

**Comment by lionelalexandre** [comment]
Dear Kurt,

the Conquest interface for ASE should be available very soon on ASE github.
Meanwhile, I can provide a local copy if you want.

Regards,
Lionel

Le mar. 28 févr. 2023 à 14:45, Kurt Irvin Rojas ***@***.***>
a écrit :

> Hi
>
> I'm currently trying to explore conquest.
>
> The documentation
> <https://conquest.readthedocs.io/en/latest/ext-tools.html#atomic-simulation-environment-ase>
> mentions that an ASE interface is available, however, I checked the current
> master branch of ASE and its still not available there.
>
> May I ask where it might be (if its already available)?
>
> Thank you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OrderN/CONQUEST-release/issues/167>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADIMB766MW77SGKPUQ4Z5YDWZX6PNANCNFSM6AAAAAAVKX3HU4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
________________________________
Dr. Lionel A. Truflandier

The CONQUEST code has been released! check at http://www.order-n.org
___
Theoretical Chemistry & Modeling Group
Institut des Sciences Moléculaires
Université Bordeaux
351 Cours de la Libération
33405 Talence, France

+33(0)5 4000 2794
***@***.*** ***@***.***>
***@***.***

**Comment by kimrojas** [comment]
Dear Lionel, 

Thank you for the reply. 
Yes. I would appreciate a local copy. 

Best regards, 
Kurt

**Comment by kimrojas** [comment]
@lionelalexandre Hi, may I follow up on this ? Thank you.
